pkg/attestation/crafter/crafter.go

```diff
@@ -427,6 +427,13 @@ func (c *Crafter) ResolveEnvVars(ctx context.Context, attestationID string) erro
 	return nil
 }
 
+func (c *Crafter) ResolveHostedRunnerContext(ctx context.Context) {
+	c.Logger.Debug().Bool("isHosted", c.Runner.IsHosted(ctx)).Msg("is runner hosted")
+	c.CraftingState.InputSchema.IsHostedRunner = c.Runner.IsHosted(ctx)
```

Comment:
This is what I mean with the comment above. `CraftingState` is the object holding the current state. `InputSchema` is supposed to be immutable.

Reply:
As mentioned in the other comment - done.